tests/test_diaCalculationPlugins.py

```diff
@@ -872,6 +880,76 @@ def testCalculate(self):
                                np.sqrt(1 / (n_sources - 1)))
 
 
+class TestMultiLombScarglePeriodogram(unittest.TestCase):
+
+    def testCalculate(self):
+        """Test Mulitband Lomb Scargle Periodogram."""
+        n_sources = 10
+        objId = 0
+
+        # Create synthetic multi-band data
+        times = np.linspace(0, 2*np.pi, n_sources)
+        fluxes = np.sin(0.3 * times)
+        diaObjects = pd.DataFrame({"diaObjectId": [objId]})
+        diaSources = pd.DataFrame(
+            data={"diaObjectId": n_sources * [objId],
+                  "band": n_sources * ["u"],
+                  "diaSourceId": np.arange(n_sources, dtype=int),
+                  "midpointMjdTai": times,
+                  "psfFlux": fluxes,
+                  "psfFluxErr": np.zeros(n_sources)})
+
+        plug = LombScarglePeriodogramMulti(LombScarglePeriodogramMultiConfig(),
```

Comment:
I know it's what was used elsewhere in this, but please name this `plugin` (and the same below). I have no idea why Chris abbreviated it like this. In fact, I'd prefer to rename all of these in this file, but I won't ask you to do that (but I'd be happy if you did!).